lib/private.h

```diff
@@ -169,6 +169,8 @@ struct vfu_ctx {
     size_t                  client_max_data_xfer_size;
 
     struct migration        *migration;
+    bool                    migr_trans_pending;
+    vfu_msg_t               *migr_trans_msg;
```

Comment:
maybe this is in the followup but I'd prefer "pending_msg" or something?

Reply:
We do rename it to `pending_msg` in https://github.com/nutanix/libvfio-user/pull/609.

Reply:
sure